agent/resources.go

```diff
@@ -0,0 +1,25 @@
+package agent
+
+import ()
+
+type Resources struct {
+	CpuTotal    uint64 `json:cpu_total`
+	CpuUsage    uint64 `json:cpu_usage`
+	MemoryTotal uint64 `json:memory_total`
+	MemoryUsage uint64 `json:memory_usage`
+	DiskTotal   uint64 `json:disk_total`
+	DiskUsage   uint64 `json:disk_usage`
+}
```

Comment:
The code refers to https://github.com/quxiao/mc/blob/master/memory.go#L12 and https://github.com/c9s/goprocinfo/blob/master/linux/cpuinfo.go#L55 . We may change to interface when we find an better way to do that. Merge this right now and you can change it anytime you want :smiley: